src/lib/components/Tooltip.svelte

```diff
@@ -29,6 +29,10 @@
       return;
     }
 
+    if (isNullish(text) || text.length === 0) {
```

Comment:
Not really important, but this could be a nice local function like `const notText  = () => isNullish(text) || text.length === 0;` Or even a variable.

Reply:
We have a utils `notEmptyString`.

Reply:
Thanks. Done.